slackbridge/configs.py

```diff
@@ -0,0 +1,9 @@
+def init(conf):
+    global slack_token, slack_uid
+    global nickserv_pass
+    global imgur_id
```

Comment:
I think a better pattern than making these global would be to have a dictionary or something that can be imported from this file, or perhaps just have this method return that config dictionary.